packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -129,10 +132,12 @@ export class Listbox extends FASTElement {
     /**
      * @internal
      */
-    protected setDefaultSelectedOption(): void {
-        let selectedIndex = this.options.findIndex(el => el.selected);
-        selectedIndex = selectedIndex !== -1 ? selectedIndex : 0;
-        this.setSelectedOption(selectedIndex);
+    protected async setDefaultSelectedOption() {
+        const selectedIndex = await Promise.resolve(
```

Comment:
Seems like the Promise isn't needed here and this could be made into a non-async method.

Reply:
This had a strong code smell when I put it in, and your comment led me towards fixing a lot of async-ish happenings throughout listbox and select.